### PR TITLE
Improve CDH docs

### PR DESCRIPTION
**Pull request by wainersm**
Sending a few improvements to the CDH docs:

* CDH: document the configuration heuristic
* CDH: document resources services

Cc @fitzthum @Xynnn007

### CODE REVIEW

**Review by Xynnn007 — APPROVED** [review]
Thanks @wainersm  for this guide. I think the link ci error will be green once this merged.

File: confidential-data-hub/README.md [hunk 1]
```diff
@@ -65,3 +65,16 @@ confidential-data-hub -c <path-to-config>
...
 Please see the [example config file](./example.config.toml) for more details.
+
+However, if a file isn't passed with **-c** then it will search for configurations on the
+following locations (in order):
+
+* **/etc/confidential-data-hub.toml**
+* **AA_KBC_PARAMS** environment variable
+* **agent.aa_kbc_params** parameter from the Kernel command-line (`/proc/cmdline`)
+
+There is a special case which is when running from [peer pods](http://github.com/confidential-containers/cloud-api-adaptor). It
```
Comment (Xynnn007) [inline confidential-data-hub/README.md:76]: Should this be `https`?

Comment (wainersm) [reply confidential-data-hub/README.md:76]: I will changed that and the other occurrences , thanks @Xynnn007 

The checker for links has failed on a relative link:
```
[ERR] [file:///home/runner/work/guest-components/guest-components/confidential-data-hub/docs/(../README.md#configuration-file)](file:///home/runner/work/guest-components/guest-components/confidential-data-hub/docs/(../README.md#configuration-file)) | Failed: Cannot find file
```

VSCode is able to follow the link...anyway, I will fix that.

File: confidential-data-hub/docs/RESOURCES_SERVICES.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      command:
+        - sh
+        - -c
+        - |
+          curl http://127.0.0.1:8006/cdh/resource/default/key/1; tail -f /dev/null
+```
+
+## Example: offline_fs_kbc
+
+In this example we will use the `offline_fs_kbc` KBC to get resouces from the local filesystem.
```
Comment (fitzthum) [inline confidential-data-hub/docs/RESOURCES_SERVICES.md:55]: Maybe mention here that this example is not running with k8s or a VM or anything. Also minor typo `resouces` -> `resources`.

File: confidential-data-hub/README.md [hunk 3]
```diff
@@ -65,3 +65,16 @@ confidential-data-hub -c <path-to-config>
...
+will try to read from the kata-agent file (**/etc/agent-config.toml** or **KATA_AGENT_CONFIG_PATH** environment variable) pior
```
Comment (fitzthum) [inline confidential-data-hub/README.md:77]: nit: `pior` -> `prior`.

**Review by fitzthum — APPROVED** [review]
Two nits but otherwise looks good

File: confidential-data-hub/README.md (hunk 1 above)
Comment (wainersm) [reply confidential-data-hub/README.md]: Fixed!

File: confidential-data-hub/docs/RESOURCES_SERVICES.md (hunk 2 above)
Comment (wainersm) [reply confidential-data-hub/docs/RESOURCES_SERVICES.md:55]: good idea

File: confidential-data-hub/README.md (hunk 3 above)
Comment (wainersm) [reply confidential-data-hub/README.md:77]: curiosity: PT_BR("pior") == EN_US("worst")

**Comment by wainersm** [comment]
addressed @fitzthum 's comments

**Pull request merged**